### PR TITLE
Refactor packages

### PR DESCRIPTION
**Pull request by pleshakov**
### Proposed changes
```    
Refactor nginx package:

    - Rename the nginx package to configs
    - Extract the nginx package from the configs package
    - Export less in the nginx package

Refactor controller package:

    - Embed the utils package into the controller package
    - Rename the controller package to k8s
    - Embed the handlers package into the k8s package
    - Move handlers initialization from the main func to NewLoadBalancerController
    - Export less in the k8s package
```

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master

### CODE REVIEW

File: internal/k8s/controller_test.go [hunk 1]
```diff
@@ -8,9 +8,9 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/nginxinc/kubernetes-ingress/internal/configs"
 	"github.com/nginxinc/kubernetes-ingress/internal/nginx"
-	"github.com/nginxinc/kubernetes-ingress/internal/nginx/plus"
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
```
Comment (Rulox) [inline internal/k8s/controller_test.go]: What version of go did you use for this? My `go-fmt` still adds `v1` automatically here 🤔

Comment (Dean-Coakley) [reply internal/k8s/controller_test.go]: Mine doesn't anyway - `go version go1.12 darwin/amd64`

Comment (Rulox) [reply internal/k8s/controller_test.go]: Weird

**Comment by boxer3379** [comment]
It works

**Pull request merged**